jwt/api.py

```diff
@@ -16,7 +16,7 @@
 
 
 class PyJWT(object):
-    def __init__(self, algorithms=None):
+    def __init__(self, algorithms=None, options=None):
```

Comment:
Grouping arguments into a category is better than having a lot of uncategorized arguments. If there are more unrelated  types of arguments in the future, they could be added to another group.